### ISSUE TITLE
Add a picker for YouTube multi-language audio

### ISSUE THREAD

**Issue by OfficialCRUGG** [imputnet/cobalt#347]
**describe the feature you'd like to see**
When downloading a YouTube video that has [multiple language tracks](https://support.google.com/youtube/answer/13338784?hl=en) (either the full video, or just the audio), I'd like to be able to use a different language audio track than the default.

**describe alternatives you've considered**
/

**additional context**
An example of such a video would be https://www.youtube.com/watch?v=mImFz8mkaHo

**Comment by wukko** [comment]
hey! it’s already supported, check settings.

i assume you want a picker instead of using your browser language, so i’ll rename the issue to that

**Comment by OfficialCRUGG** [comment]
Ah, I see. Yeah a picker of some sort would be great. My OS & browser are set to English, so this usually just ends up being the default language.

**Comment by wukko** [comment]
you can temporarily move the preferred sub language over english in browser's language settings

**Comment by OfficialCRUGG** [comment]
Good idea, that'd work for the time being. A language picker would still be more ideal for the future though, I'd say.

**Comment by wukko** [comment]
hey! this was added a while ago, you can find the language picker here: https://cobalt.tools/settings/audio#youtube-dub

**Comment by Libellule1971** [comment]
Would it be easier when I set up «  preferred dub language : original », to avoid dub language ?
This option doesn’t work when i use my VPN not in local area.

**Comment by wukko** [comment]
when the dub language is set to original, the original audio track is used, aka no dub

**Comment by wukko** [comment]
your ip geolocation doesn’t affect cobalt, because requests to youtube are done on our servers
